rpcs3/Emu/RSX/GL/GLHelpers.h

```diff
@@ -133,7 +133,7 @@ namespace gl
 			std::string vendor_string = (const char*)glGetString(GL_VENDOR);
 			std::transform(vendor_string.begin(), vendor_string.end(), vendor_string.begin(), ::tolower);
 
-			if (vendor_string.find("intel"))
+			if (vendor_string.find("intel") != std::npos)
```

Comment:
Thanks. Its a typo. I put these edits together from notepad and didn't check too carefully. The flow control ones I almost accidentally deleted since they've been sitting in my repo for some time and I just forgot about them.